people/bradsmith.json

```diff
@@ -2,19 +2,66 @@
+    },
+    {
+      "sitename": "Study Computers",
+      "siteurl": "https://web.archive.org/web/20111104104241/http://www.studycomputers.com/"
+    },
+    {
+      "sitename": "Brad the Nurse",
+      "siteurl": "https://web.archive.org/web/20120617165745/http://www.bradthenurse.com/"
     }
   ],
-  "contributions": [],
-  "gallery": []
+
+  "contributions": [
+    {
+      "title": "2014 - Presentation - Hacker Halted - Cracking BlueTooth for Phun and Profit",
+      "url": "  https://www.yumpu.com/en/document/view/22139939/cracking-bluetooth-for-phun-and-profit-brad-smith-hacker-halted",
+      "description": ""
+    },
+    {
+      "title": "2010 - Video - DEF CON 18 - Weaponizing Lady GaGa, Psychosonic Attacks",
+      "url": "https://youtu.be/hD8xBAEsIV8?si=51ZWgmbpSTvaAieX",
+      "description": "This session introduces and demonstrates the emerging attack vector of psychosonics. Attend and you'll understand how to turn ANY MP3 into a weapon, a study aid, a hidden calming session or helping you experience that Ah-Ha moment of discovery simply by injecting an alternate data stream attack made up of psychosonic frequencies. You'll learn how different mental states can be created using frequencies that interact with the brain, how the military is using this attack vector, how Vegas uses these same techniques on customers, which open source software creates these frequency generated psychic states and sites so you can continue your adventures in psychosonics. Multiple new attacks based on psychosonics will be demonstrated and fully explained to you can easily integrated these into you attack tools. This is an 'attack the audience' session where you'll actually experience these psychosonic attacks so you can judge their effectiveness for yourself. Better yet, you'll understand how to incorporate this attack vector into your future attack surface. Hey, psychosonics is much better than the flame thrower bra she already has!"
+    },
+    {
+      "title": "2006 - Video - DEF CON 14 - How the FBI uses NLP on YOU!",
+      "url": "https://youtu.be/KSvGgLXQF8U?si=59qwyPGuZarEmmot",
+      "description": "This session will reveal to you how the FBI uses Neuro-Linguistic Programming (NLP) during interview and interrogation sessions. Gaining cooperation with special speech and word changes, Clues to help determine whether clients are lying or remembering and the traditional 'Cop Stop technique' will all be revealed and practiced by attendees. Seldom taught outside the law or medical community, you'll be instructed in and actual practice techniques, just like the Feds do. While you may not be able to stop leaking information to the specially trained professional, you'll now see the extra information other are giving off. Come prepared to talk to others and learn invaluable skills that can C.Y.A."
+    }
+  ],
+  "gallery": [
+    {
+      "url": "/images/bradsmith01.jpg",
```

Comment:
This image appears to actually be /images/bradsmith.jpg

Reply:
Nice catch; apologies for the oversight. Changes made and pushed up!